code/libraries/MailgunLib/MailgunLib.js

```diff
@@ -1,24 +1,40 @@
 /**
  * Mailgun - Send emails via Mailgun REST API
- * @typedef Mailgun
- * @param {string} apikey API Key found in Mailgun Console (https://www.mailgun.com/)
- * @param {string} domain Domain ex. mg.yoursite.com, provided by Mailgun
- * @param {string} originEmail origin address from which to send emails, provided by Mailgun
+ * https://documentation.mailgun.com/en/latest/api_reference.html
+ * 
+ * @typedef {Object} Mailgun
+ * @param {string} apiKey - API Key found in Mailgun Console (https://www.mailgun.com/)
+ * @param {string} domain - Domain ex. mg.yoursite.com, provided by Mailgun
+ * @param {string} originEmail - origin address from which to send emails, provided by Mailgun
  */
  function Mailgun(apiKey, domain, originEmail){
      
      if(!apiKey || !domain || !originEmail){
          throw new Error("Invalid configuration - must provide apiKey, domain, and originEmail. Can be acquired on Mailgun account home.")
      }
      /**
+      * This method is used to send an email with subject and body and other components can be added. Refer to the mailgun api https://documentation.mailgun.com/en/latest/api-sending.html#sending
+      * 
       * @memberof Mailgun
       * @param {string} body
```

Comment:
these params need descriptions and examples

Reply:
Added description, an example is there right below it.